core/arch/arm/kernel/early_ta.c

```diff
@@ -16,14 +16,14 @@
 #include <zlib.h>
 
 struct ts_store_handle {
-	const struct early_ta *early_ta;
+	const struct embedded_ts *early_ta;
 	size_t offs;
 	z_stream strm;
 };
 
-static const struct early_ta *find_early_ta(const TEE_UUID *uuid)
+static const struct embedded_ts *find_early_ta(const TEE_UUID *uuid)
 {
-	const struct early_ta *ta;
+	const struct embedded_ts *ta;
```

Comment:
maybe also add an init value. (could be made in a later P-R) (here and occurrences below)

Reply:
done